### PR TITLE
Remove classic tickers

### PR DESCRIPTION
**Pull request by adamdossa**
### modified API

- Removes storage items related to classic tickers

### data migration

- Removes storage related to classic tickers during upgrade

### CODE REVIEW

File: pallets/asset/src/lib.rs [hunk 1]
```diff
@@ -329,22 +302,8 @@ decl_storage! {
         StorageVersion get(fn storage_version) build(|_| Version::new(1)): Version;
```
Comment (Neopallium) [inline pallets/asset/src/lib.rs]:
```suggestion
        StorageVersion get(fn storage_version) build(|_| Version::new(2)): Version;
```

Comment (Neopallium) [reply pallets/asset/src/lib.rs]: Need to bump that version to match the newest storage version of the pallet.

File: pallets/asset/src/lib.rs [hunk 2]
```diff
@@ -372,6 +331,15 @@ decl_module! {
         const AssetMetadataValueMaxLength: u32 = T::AssetMetadataValueMaxLength::get();
         const AssetMetadataTypeDefMaxLength: u32 = T::AssetMetadataTypeDefMaxLength::get();
 
+        // Remove all storage related to classic tickers in this module
+        fn on_runtime_upgrade() -> Weight {
+            use polymesh_primitives::storage_migrate_on;
+            storage_migrate_on!(StorageVersion, 1, {
```
Comment (Neopallium) [inline pallets/asset/src/lib.rs]:
```suggestion
            storage_migrate_on!(StorageVersion, 2, {
```

Comment (Neopallium) [reply pallets/asset/src/lib.rs]: This version number here is the target version, which is version `2` now.  We removed the old migrations, which is why we don't have one for version `1` anymore.

**Review by Neopallium — APPROVED** [review]

**Review by HenriqueNogara — APPROVED** [review]

**Pull request merged**